### PR TITLE
Remove deprecated methods

### LINKED ISSUE #945: Remove deprecated methods like GraphHopper.setWeighting(s)

### PR DESCRIPTION
**Pull request by karussell**
Fixes #945 

This breaks the old API but was announced earlier via deprecated.

### CODE REVIEW

File: core/files/changelog.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 0.9
-    
+    removed deprecated methods in GraphHopper, Path.calcMillis, LocationIndex.findID
```
Comment (boldtrn) [inline core/files/changelog.txt]: Should we also name the removed methods in GraphHopper?

File: core/src/main/java/com/graphhopper/storage/index/LocationIndex.java [hunk 2]
```diff
@@ -42,12 +42,6 @@
     LocationIndex prepareIndex();
 
     /**
-     * @return the closest node id for the specified geo location (latitude,longitude)
-     * @deprecated will be removed with 0.8 use 'QueryResult findClosest' instead
```
Comment (boldtrn) [inline core/src/main/java/com/graphhopper/storage/index/LocationIndex.java]: BTW: Why haven't we marked the method with @Deprecated. Maybe someone continued to use them without reading the comment?

Comment (karussell) [inline core/src/main/java/com/graphhopper/storage/index/LocationIndex.java]: Not sure. For my IDEs it does not matter if I mark them in comment or outside.

File: core/files/changelog.txt (hunk 1 above)
Comment (karussell) [inline core/files/changelog.txt]: ok

**Comment by karussell** [comment]
It should me shown in IntelliJ too: 
deprecated

**Comment by karussell** [comment]
Ah, no. That is only the interface and for any implementation it is not shown!? But this does not change with the explicit annotation. Strange. This smells like a bug to me :)

**Comment by boldtrn** [comment]
> Ah, no. That is only the interface and for any implementation it is not shown!? But this does not change with the explicit annotation. Strange. This smells like a bug to me :)

Ah, I only tried it for one implementation. So in this case I will approve this PR. Makes no sense to block this PR only because of buggy intelliJ implementation ;).

**Comment by karussell** [comment]
BTW: it is not a bug, it is a (configuration) feature ...
deprecated

**Pull request merged**